catalogue/webapp/pages/PROGRESS_NOTES.md

```diff
@@ -8,17 +8,18 @@ For a more comprehensive search of all of our collections, [wellcomelibrary.org]
 
 We will keep this page up to date with the latest information as we make more progress.
 
-## What’s new? (updated 20 November 2019)
+## What’s new? (updated 8 July 2020)
 
 ### Improvements to search relevance
 
-#### Asking for feedback directly
-We’re asking for your input on the relevance of the search results we're showing you. By opting to rate the results you see when you search, you’ll be letting us know not only how relevant they are, but what a more relevant set or ordering might look like. We’ll use this feedback to continue fine-tuning the algorithms that search our collections and return results to you.
-
 #### Fine tuning relevance
-Based on the data we already have about how search is being used, we've tested a change to the way queries are split and combined. Rather than letting results with _any_ (one or more) of the terms appear in your results (using the OR operator between query tokens), we now require that _all_ of the tokens appear in any result you see (using the AND operator between query tokens). This seems to be returing more relevant results, so we've made this the default algorithm, while kicking off a new test.
+Based on the data we already have about how search is being used, we've tested a change to the way queries are split and combined. Rather than letting results with _any_ (one or more) of the terms appear in your results (using the OR operator between query tokens), we now require that _all_ of the tokens appear in any result you see (using the AND operator between query tokens). This seems to be returning more relevant results, so we've made this the default algorithm, while kicking off a new test.
+
+## What's in development now
+- *Search quality*: Based on feedback from library catalogue users, we are now working to improve Title and Author searches. We're continuing to monitor 
```

Comment:
Is this sentence supposed to end so abruptly?

Reply:
nope, removed it.